shell/server/accounts/ldap/ldap-server.js

```diff
@@ -23,7 +23,7 @@ LDAP_DEFAULTS = {
  @class LDAP
  @constructor
  */
-let LDAP = function () {
+LDAP = function () {
```

Comment:
I'm open to suggestions on what to do then. We've tried to move away from using packages, which is the only other reasonable way I know how to do this.